### DISCUSSION TITLE
An Idea I had

### DISCUSSION THREAD

**Discussion by Shamil-FD** [Shamil-FD/ToastyXD#8] [General]
Here's an idea I had for the bot but I gave up on it. Just posting it here so you can get inspired by this and make this or similar stuff to your bot

> Hallo there! This is an important question that I need all of you to answer. So, I was thinking about making an economy system for this server, but my plan for thy system is enormous, bigger than your dic-.. bigger than you would think... So this is how the system would be like 

 >   - It will be minecraft themed they would mine all day... mine all night.. mine until-

>    - There would be a shop with some things like 

>    1. A custom role

 >   2. Badges 

 >   etc..

>    - They would be able to trade and sell stuff

 >   - There would be zombie attacks

>    - They would be able to craft things that have much better value

 >   - Minecraft Girlfriends wink

 >   - So much more

> Have an amazing rest of your day!

**Comment by FC5570** [comment]
sounds fun
